tutorials-book/src/ch01-02-migration-cli.md

```diff
@@ -0,0 +1,213 @@
+    fn name(&self) -> &str {
+        "m_20220101_000002_create_baker_table" // Make sure this matches with the file name
+    }
+}
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    // Define how to apply this migration: Create the Baker table.
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .create_table(
+                Table::create()
+                    .table(Baker::Table)
+                    .col(
+                        ColumnDef::new(Baker::Id)
+                            .integer()
+                            .not_null()
+                            .auto_increment()
+                            .primary_key(),
+                    )
+                    .col(ColumnDef::new(Baker::Name).string().not_null())
+                    .col(ColumnDef::new(Baker::ContactDetails).json())
+                    .col(ColumnDef::new(Baker::BakeryId).integer().not_null())
+                    .foreign_key(
+                        ForeignKey::create()
+                            .name("fk-baker-bakery_id")
+                            .from(Baker::Table, Baker::BakeryId)
+                            .to(Bakery::Table, Bakery::Id),
+                    )
+                    .to_owned(),
+            )
+            .await
+    }
+
+    // Define how to rollback this migration: Drop the Baker table.
+    async fn down(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .drop_table(Table::drop().table(Baker::Table).to_owned())
+            .await
+    }
+}
+
+// For ease of access
+#[derive(Iden)]
+pub enum Baker {
+    Table,
+    Id,
+    Name,
+    ContactDetails,
+    BakeryId,
+}
+```
+
+```rust, no_run
+// migration/src/lib.rs
+
+pub use sea_orm_migration::prelude::*;
+
+// Add each migration file as a module
+mod m20220101_000001_create_bakery_table;
+mod m20220101_000002_create_baker_table;
+
+pub struct Migrator;
+
+#[async_trait::async_trait]
+impl MigratorTrait for Migrator {
+    fn migrations() -> Vec<Box<dyn MigrationTrait>> {
+        vec![
+            // Define the order of migrations.
+            Box::new(m20220101_000001_create_bakery_table::Migration),
+            Box::new(m20220101_000002_create_baker_table::Migration),
+        ]
+    }
+}
+```
+
+**Important**: Make sure the following features are enabled in the `migration` crate. The database driver feature must match the database being used.
+
+```diff
+# migration/Cargo.toml
+
+...
+
+[dependencies.sea-orm-migration]
+version = "^0.8.0"
+features = [
++   "sqlx-mysql",
++   "runtime-async-std-native-tls",
+]
+```
+
+## Perform the migrations
+
+Perform all the migrations through `sea-orm-cli`:
+
+```sh
+# Change the value of DATABASE_URL according to your database implementation.
+# Make sure the database name is also supplied for MySQL or PostgreSQL.
+$ DATABASE_URL="mysql://root:root@localhost:3306/bakeries_db" sea-orm-cli migrate refresh
+```
```

Comment:
It's better to tell reader to execute this command on the project root
- https://www.sea-ql.org/SeaORM/docs/migration/running-migration#via-sea-orm-cli